src/openforms/submissions/templates/admin/submissions/submission/change_form.html

```diff
@@ -21,5 +21,24 @@
                 </div>
             </div>
         </div>
+        <div class="form-row field-attachments">
+            <div>
+                <label>{% trans "Attachments:" %}</label>
+                <div class="readonly">
+                    <ul>
+                        {% for key, files in attachments.items %}
+                            <li>{{ key }}:
+                                {% for file in files %}
+                                    <a href="{{ file.content.url }}">
```

Comment:
I suspect this will be a URL that doesn't work because of the private media field thing, unless that magically uses the `PrivateMediaView`.

Reply:
I swapped the admin-side file views for custom views.